de-DE/lang/numere.nlng

```diff
@@ -1175,6 +1178,10 @@ GUI_EDITOR_CALLTIP_CONST_GAMMA_E_[PARTICLE]=_gamma_e = 1.760859644e11 [1/(T s)]
 GUI_EDITOR_CALLTIP_CONST_LANDE_E_[PARTICLE]=_lande_e = -2.00231930436182
 GUI_EDITOR_CALLTIP_CONST_FEIGENBAUM_DELTA_[MATHEMATICAL]=_feigenbaum_delta = 4.6692016091
 GUI_EDITOR_CALLTIP_CONST_FEIGENBAUM_ALPHA_[MATHEMATICAL]=_feigenbaum_alpha = 2.5029078751
+GUI_EDITOR_CALLTIP_CONST_DAY_SECS_[MATHEMATICAL]=_day_secs = 86400
+GUI_EDITOR_CALLTIP_CONST_HOUR_SECS_[MATHEMATICAL]=_hour_secs = 3600
+GUI_EDITOR_CALLTIP_CONST_WEEK_SECS_[MATHEMATICAL]=_week_secs = 604800
+GUI_EDITOR_CALLTIP_CONST_YEAR_SECS_[MATHEMATICAL]=_year_secs = 31536000

```

Comment:
It would be nice to have another category for those constants like `TIME`. This can be achieved by modifying the `main.nlng` files in all languages as follows:
- Add a new key below `PARSERFUNCS_LISTCONST_TYPE_INTERNAL` as `PARSERFUNCS_LISTCONST_TYPE_TIME` with a translation
- Add `time` to the keylist in `GUI_TREE_CONST_KEYLIST`